.github/stale.yml

```diff
@@ -0,0 +1,16 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 90
+# Never close the issues
+daysUntilClose: false
+# Issues with these labels will never be considered stale
+# Only exempting feature requests right now as they are picked by product
+exemptLabels:
+  - kind/feature
```

Comment:
Let's switch this to only consider `kind/bug` (and issues with no `kind/*` label if possible) for now and expand later if useful.

Reply:
I have added all other kind labels except `kind/bug` in the exempt list. This way it will be possible to have `issues with no kind/* label if possible`. There is `onlyLabels` options but that won't allow `issues with no kind/* label if possible`